CHANGELOG.md

```diff
@@ -8,3 +8,13 @@ Converts VCF variants into a [FHIR Genomics Diagnostic Report](http://hl7.org/fh
 * Supports genome build ('GRCh37' or 'GRCh38');
 * Optionally supports a query region in the form of .bed or dictionary that indicates the region(s) to convert.
 * Optionally supports inclusion of  'region-studied' observations that detail which portions of the conversion region were studied, and of those studied regions, which portions were deemed uncallable.
+
+# v0.0.17 (2021-06-07)
+
+## Added
+* Exposed a new parameter to configure ratio for determining Homoplasmic vs Heteroplasmic.
+
+## Fixed
+* Excluded records with incorrect CHROM values in vcf file ([#19](https://github.com/elimuinformatics/vcf2fhir/issues/19)).
+* Removed 'Human reference sequence assembly version' component from variant observation ([#36](https://github.com/elimuinformatics/vcf2fhir/issues/36)).
+* Case where conversion region is not studied ([#66](https://github.com/elimuinformatics/vcf2fhir/issues/66)).

```

Comment:
The latest version should come first. Can you please move this before `0.0.1 - 0.0.16`